source/common/http/path_utility.h

```diff
@@ -24,5 +29,21 @@ class PathUtil {
   static absl::string_view removeQueryAndFragment(const absl::string_view path);
 };
 
+using Transformation = std::function<absl::optional<std::string>(absl::string_view)>;
+
+class PathTransformer {
+public:
+  PathTransformer(envoy::type::http::v3::PathTransformation operations);
+
+  absl::optional<std::string> transform(const absl::string_view original_path);
+
+  static absl::optional<std::string> mergeSlashes(absl::string_view original_path);
+
+  static absl::optional<std::string> rfcNormalize(absl::string_view original_path);
+
+private:
+  std::list<Transformation> transformations;
```

Comment:
style: member should end with _ (`transformations_`)

Reply:
Thanks.